war/src/main/js/recommendedPlugins.js

```diff
@@ -0,0 +1,125 @@
+module.exports = {
+	"defaultPlugins": [ "workflow-aggregator", "github" ],
+	"availablePlugins": [
+		{
+			"category": "General",
+			"description": "(a collection of things I cannot think of a better name for)",
+			"plugins": [
+			  { "name": "ansicolor" },
+			  { "name": "build-timeout" },
+			  { "name": "embeddable-build-status" },
+			  { "name": "envinject" },
+			  { "name": "next-build-number" },
+			  { "name": "rebuild" },
+			  { "name": "sidebar-link" },
+			  { "name": "simple-theme-plugin" },
+			  { "name": "timestamper" },
+			  { "name": "translation" },
+			  { "name": "ws-cleanup" }
+			]
+		},
+		{
+			"category":"Organization and Administration",
+			"plugins": [
+			  { "name": "antisamy-markup-formatter" },
+			  { "name": "cloudbees-folder" },
+			  { "name": "compact-columns" },
+			  { "name": "dashboard-view" },
+			  { "name": "disk-usage" },
```

Comment:
given disk-usage doesn;t scale very well should disk-usage-simple (or whatever it is called) be in here as well?